packages/react-native-editor/bin/generate-pot-files.sh

```diff
@@ -151,7 +152,7 @@ function generate_pot_files() {
   mkdir -p $output_path
 
   if [ -n "$subtract_pot_files" ]; then
-    echo "--- Strings from ${plugins_to_subtract[@]} plugins will be subtracted ---"
+    echo "-- Strings from ${plugins_to_subtract[@]} plugins will be subtracted --"
```

Comment:
The value `---` is used by Buildkite to group log output ([reference](https://buildkite.com/docs/pipelines/managing-log-output#collapsing-output)), so I made this change in order to prevent grouping it.